faqs/galaxy/support_messaging_system.md

```diff
@@ -0,0 +1,108 @@
+---
+title: How can I use the Matrix messaging system for Galaxy Project communication?
+area: support
+description: When you are directed to use Matrix for a Galaxy working group or for your support question, but you don't know what Matrix or how to access it. Learn what Matrix is, why it is used, and how to use Matrix to connect to us.
+box_type: tip
+layout: faq
+contributors: [hujambo-dunia]
+---
+
+### Introduction
+
+The Galaxy community uses [Matrix](https://matrix.org/) – a secure, decentralized, open-source messaging protocol – as its primary chat system. Matrix enables real-time communication across Galaxy contributors, developers, trainers, and users. Common uses include:
+
+* Getting help from the community
+* Discussing tool and workflow development
+* Organizing training events and materials
+* Collaborating on Galaxy infrastructure or scientific questions
+
+Matrix provides an open alternative to proprietary platforms like Slack and Microsoft Teams, with broader flexibility and user privacy in mind.
+
+---
+
+### 🔰 For Newbies: Getting Started with Matrix
+
+If you are new to messaging systems, here's how to get started with Galaxy on Matrix:
+
+<img src="images/support_matrix_element.jpg" alt="Galaxy Project Matrix space using the Element client app" width="350" align="right">
```

Comment:
Since FAQs can be embedded anywhere, we have to provide the full path to the image:

```suggestion
<img src="{% link faqs/galaxy/images/support_matrix_element.jpg %}" alt="Galaxy Project Matrix space using the Element client app" width="350" align="right">
```